test.rb

```diff
@@ -117,13 +118,6 @@ def duplicate_measure(old_measure_dir, measure_dir, name, class_name, taxonomy_t
   #puts info
 end
 
-measures.each do |measure|
```

Comment:
@nllong I added this here to test the measure caching behavior.  This script is a test script, in a real rake task or whatever you would not call this twice